vtex/faststore
# feat: add fuzzy param as optional

## What's the purpose of this pull request?

adds the fuzzy parameter as an option so that the search result is more precise if the store needs it

## How it works?

Basically the search api already accepts this parameter in the api, I added the same parameter to the faststore queries so stores can pass the route if they want

## How to test it?

Access a search page or category by passing the parameter fuzzy=0 or fuzzy=1 and check if the result has changed
